test/test_page.rb

```diff
@@ -24,6 +24,10 @@
     assert_equal 'Bilbo-Baggins.md', page.path
     assert_equal :markdown, page.format
     assert_equal @wiki.repo.commits.first.id, page.version.id
+
+    assert_not_nil page.last_version
+    assert_equal page.versions.first.id, page.last_version.id
+    assert page.last_version.stats.files.map{|file_stats| file_stats[0]}.include? page.path
```

Comment:
Or we could refactor `#build_stats` such that files are also structs:
```ruby
files << OpenStruct.new(path: patch.delta.new_file[:path].force_encoding("UTF-8"), deletions: new_deletions, additions: new_additions, changes: patch.changes)
```
Not sure how much other stuff that would break, but that way we could do `files.map{ | file | file.path }`.